### ISSUE TITLE
The “Unable to connect” text can be easily missed

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5217]
### Is it an issue related to Adaptive Cards?

No.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA, Windows: Chrome/Firefox with JAWS

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=true

### How to reproduce the issue?

1. Open [https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=true](https://wwwppe.microsoft.com/en-us/surface) in Edge. 
2. Start NVDA. 
3. [Tab] to the ‘Help me choose’ button and open the window. Focus will be on the close “X”. 
4. Tab (x2). 
5. Use the down arrow key to access the message content. 
6. Listen to the prompt. 
7. Make a selection. 
8. Select ‘Next’. 
9. Did you hear that it was unable to connect?

### What do you expect?

The “Unable to connect” should be announced by screen reader as soon as user starts interacting with the page.
If the ‘Help me choose’ window is not “connected”, then ensure that it is provide first in the read order, and is narrated first in the read order.  

### What actually happened?

The “Unable to connect” text can be easily missed. It is only narrated when the user moves past the ‘Next’ button. 

Observation: When the ‘Next’ button is pressed it is narrated as ‘pressed’, but there is no indication that nothing happens. The user has to explore outside of that area to see if they can determine why the interface is not working.  

### Did you find any DOM elements that might have caused the issue?

![image](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/0092c894-62a8-4acb-a10e-1c7e9a5b6134)

### MAS reference

https://aka.ms/MAS1.3.1

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#AILimited;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by OEvgeny** [comment]
WebChat should now announce the status change when not connected.
